docs-src/src/layouts/Navigation.js

```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React, {useState} from 'react'
 import Menu from 'harmonium/lib/Menu'
 import Drawer from 'harmonium/lib/Drawer'
 import packageInfo from '../../../package.json'
+import Input from 'harmonium/lib/Input'
+import menuItems from '../pages/menuItems'
 
 export default function Navigation() {
+
+  //Use local state for controlled form
+  const [formValue, setFormValue] = useState('')
+
+  //Set local state to form value
+  const handleFormChange = (e) => {
+    setFormValue(e.target.value)
+  }
+
+  //Check if item is including in search form
+  function itemCheck(item){
+    return item.label.toLowerCase().includes(formValue.toLowerCase());
+  }
+
+  //Display filtered menu items 
+  function displayMenu(menu){
+    return menu.map( (cat, i) => {
+      if (cat.contents.filter( (item) => itemCheck(item)).length >= 1)
+      return ( 
+        //Category name
+        <Menu vertical key={i}>
```

Comment:
same here with the key as the index. Should use `cat.label` or the like. Here's a real old blog post about this https://revelry.co/resources/development/dynamic-child-components-with-react/